mantidproject/mantid
# PSI muon data loader

**Description of work.**
I have created an algorithm that will load in data from a .bin file from the PSI Muon facility in Switzerland, and create a workspace utilizing the data in these files as a histogram.

**Report to:**  

**To test:**
Incoming Unit Tests....

But for now you can load this file: 
[deltat_tdc_dolly_1529.txt](https://github.com/mantidproject/mantid/files/2250072/deltat_tdc_dolly_1529.txt)

Load it by renaming it to use .bin instead of .txt. Then call it in the algorithm you can see it when you load the algorithm.

Fixes #22705 

Does this update require release notes?
- [ ] To be done


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
